### PR TITLE
[T4A3][F10-B2]Ning Yu

### PR DESCRIPTION
**Pull request by ningyuuu**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/Contact.java:4]: Your comment says Contact is an abstract class. You have to use "abstract" keyword to declare a class as abstract.
This class looks more like a normal class. Yes, the purpose of inheritance can be still achieved with a normal class. :) But you need to study difference between a normal and abstract class.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+	
+
+    public String value = "";
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/Contact.java:6]: Think do you really need this to be public

File: src/seedu/addressbook/data/person/Address.java [hunk 3]
```diff
@@ -6,15 +6,12 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address extends Contact {
 
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/Address.java:14]: Do you need these constants to be public?

File: src/seedu/addressbook/data/person/Contact.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.person;
+
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/Contact.java:2]: Need attention on adding comments.

**Comment by KulaniT** [comment]
@ningyuuu : Overall nice work implementing inheritance. But you need to pay attention on following;

*Adding comments.
*Which access modifiers suitable?
*What is the difference between abstract and normal class?

File: src/seedu/addressbook/data/person/Contact.java (hunk 4 above)
Comment (ningyuuu) [reply src/seedu/addressbook/data/person/Contact.java:2]: Hi @KulaniT , could you give a bit more elaboration on what kind of comments I should be adding here? Sorry, might have missed this part of the lecture out. Thanks in advance.

Comment (ningyuuu) [reply src/seedu/addressbook/data/person/Contact.java:2]: Oh, I believe you are referring to header comments. Thanks, will remember it next time.

**Pull request closed without merging**